src/ansys/fluent/core/session_meshing.py

```diff
@@ -66,3 +71,130 @@ def PMFileManagement(self):
     def preferences(self):
         """Datamodel root of preferences."""
         return super(Meshing, self).preferences if not self.switched else None
+
+    def read_case(
```

Comment:
If you add it at this level, then it won't be available in `PureMeshing`.

Reply:
Right, I will update this change. Thank you.